app/views/dashboard/_discovery_widget.html.erb

```diff
@@ -1,5 +1,5 @@
 <h4 class="ca"><%= _('Discovered Host Pool') %> - <%= @current_permission %></h4>
-<% all_hosts = Host::Discovered.search_for '' %>
+<% all_hosts = Host::Discovered.includes(:model).includes(:discovery_attribute_set).search_for '' %>
```

Comment:
I know this was in the original code but why do we do `.search_for ''` and not `.scoped`?
or just 
``` ruby
 <% hosts =Host::Discovered.includes(:model).includes(:discovery_attribute_set).limit(5) %>
```